Tools/CISupport/ews-build/steps.py

```diff
@@ -5358,7 +5358,8 @@ def run(self, BufferLogObserverClass=logobserver.BufferLogObserver):
 
         if rc == SKIPPED:
             self.summary = 'Patches have no commit message'
-            return rc
+            defer.returnValue(rc)
+            return
```

Comment:
do we need this `return` after `defer.returnValue`?

Reply:
We need the early exit from the function.